src/templates/components/PaidForHeader.svelte

```diff
@@ -14,44 +14,49 @@
 	export let SeriesUrl: GAMVariable;
```

Comment:
I think as we're in a component here we shouldn't use `GAMVariable` might lead to unexpected behaviour, like changing the value in the index.svelte but it gets replaced here with the variable content.